package.json

```diff
@@ -186,7 +186,8 @@
       "lint": "eslint src --ext ts",
       "format": "prettier --write .",
       "test": "node ./out/test/runTest.js",
-      "coverage": "c8 --check-coverage --timeout 10000 npm run test"
+      "coverage": "c8 --check-coverage --timeout 10000 npm run test",
+      "ui-test": "npm run compile-tests && extest setup-and-run ./out/ui-test/**.test.js -s ../../src/ui-test/test-repo/flask-hello-world"
```

Comment:
The compile part should be a pre script.

https://docs.npmjs.com/cli/v6/using-npm/scripts#pre--post-scripts

Reply:
thanks, changed it!